iosHackthonApp/Modules/Shared/Strings/Sources/Strings/Strings.swift

```diff
@@ -9,112 +9,115 @@
 import Foundation
 
 public struct Strings { // TODO: these can all be public enums, plus public extensions will help reduce the need to add `public` everywhere
```

Comment:
I'll work on this soon, it'll make this feel less tedious now it's in a module and needs to have `public` added